pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -244,12 +250,16 @@ def from_mcp_sdk(cls, mcp_capabilities: mcp_types.ServerCapabilities) -> ServerC
         Args:
             mcp_capabilities: The MCP SDK ServerCapabilities object.
         """
+        tools_cap = mcp_capabilities.tools
+        resources_cap = mcp_capabilities.resources
         return cls(
             experimental=list(mcp_capabilities.experimental.keys()) if mcp_capabilities.experimental else None,
             logging=mcp_capabilities.logging is not None,
             prompts=mcp_capabilities.prompts is not None,
-            resources=mcp_capabilities.resources is not None,
-            tools=mcp_capabilities.tools is not None,
+            resources=resources_cap is not None,
+            tools=tools_cap is not None,
+            tools_list_changed=bool(tools_cap.listChanged) if tools_cap else False,
+            resources_list_changed=bool(resources_cap.listChanged) if resources_cap else False,
```

Comment:
Let's do this for do this for `mcp_capabilities.prompts.listChanged` as well, even if we don't support listing prompts yet